packages/node/src/rest/dtos/organizations/create-organization.dto.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  IsArray,
+  IsNotEmpty,
+  IsObject,
+  IsString,
+  IsUrl,
+} from "class-validator";
+
+import {
+  Location,
+  OrganizationMember,
+  OrganizationSocialLink,
+} from "../../types";
+
+export class CreateOrganizationDto {
+  @IsString()
+  @IsNotEmpty()
+  name: string;
+
+  @IsArray()
+  members?: OrganizationMember[];
+
+  @IsString()
+  @IsUrl({
+    protocols: ["http", "https"],
+  })
+  logoUrl?: string;
```

Comment:
logoUrl is in OrganizationIdentity embedded property

Reply:
I copied and pasted your code 🥲